### ISSUE TITLE
BrowserLink Should Use HTTPS

### ISSUE THREAD

**Issue by RehanSaeed** [aspnet/Tooling#937]
When debugging your site using HTTPS with IIS Express or dotnet run, Browser Link injects scripts from the HTTP scheme. This breaks pages, because you get mixed content security warnings. Ideally Browser Link should be using HTTPS as the default. 

Related Issue https://github.com/aspnet/Tooling/issues/490 already shows that Browser Link does not work with HTTPS at all.

**Comment by damienbod** [comment]
Is thie related:
https://github.com/aspnet/Tooling/issues/830

**Comment by jodavis** [comment]
@RehanSaeed This should be working in VS 2017 RC with the latest Microsoft.VisualStudio.Web.BrowserLink package (1.0.1 or 1.1.0). Browser Link should add an HTTPS link when the request is made against an HTTPS URL. Are you still seeing the issue with these releases?

**Comment by Whathecode** [comment]
@jodavis Yes, using Microsoft.VisualStudio.Web.BrowserLink works. However, I ended up here because I was using a ASP.NET Core Web Application template which seemingly imported Microsoft.VisualStudio.Web.BrowserLink.Loader (14.1.0) instead. Uninstalling the latter and installing the former resolved the issue for me.

**Comment by madmunsterdaddy** [comment]
I still can't get BrowserLink to work with HTTPS. It just doesn't show up at all even in VS 2017. Or is that still not fixed yet per https://github.com/aspnet/Tooling/issues/490
